bin/sl-executor-install.txt

```diff
@@ -3,19 +3,22 @@ usage: %MAIN% --control URL [options]
 Install the StrongLoop Mesh Executor as a service.
 
 Options:
-  -h,--help           Print this message and exit.
-  -v,--version        Print version and exit.
-  -b,--base BASE      Base directory to work in (default is $HOME of the user
-                      that executor is run as, see --user).
-  -C,--control URL    Connect to central at this URL (required, no default).
-  -u,--user USER      User to run executor as (default is strong-executor).
-  -P,--base-port PORT Applications run on PORT + instance ID (default 3000).
-  -n,--dry-run        Don't write any files.
-  -j,--job-file FILE  Path of Upstart job to create (default is
-                      `/etc/init/strong-executor.conf`).
-  -f,--force          Overwrite existing job file if present.
-  --upstart VERSION   Specify Upstart version, 1.4 or 0.6 (default is 1.4).
-  --systemd           Install as a systemd service, not an Upstart job.
+  -h,--help               Print this message and exit.
+  -v,--version            Print version and exit.
+  -b,--base BASE          Base directory to work in (default is $HOME of the
+                          user that executor is run as, see --user).
+  -C,--control URL        Connect to central at this URL (required, no default).
+  -u,--user USER          User to run executor as (default is strong-executor).
+  -P,--base-port PORT     Applications run on PORT + instance ID (default 3000).
+  -A,--routable-addr ADDR Address to use when routing traffic to services hosted
+                          on this executor. (default: IP of the interface
```

Comment:
I wonder if `app-addr` might be more clear? That's what it is, the addr to use to talk to the app, right? Which should be chosen to be routable. `routable-addr` is just a bit of a mouthful, and it describes the address, but not the meaning of the address. I like `app-addr` because it is very `-A`, but perhaps `-A,--svc-addr` would be more consistent in its use of "service".

Thoughts @rmg?